content/javascript/concepts/dates/terms/getFullYear/getFullYear.md

```diff
@@ -0,0 +1,27 @@
+---
+Title: 'getFullYear()' 
+Description: 'Return the current full year' 
+Subjects: 
+  - 'Web Development'
+Tags: 
+  - 'Date'
+  - 'Method'
+CatalogContent: 
+  - 'paths/web-development'
+---
+
+The **getFullYear** method returns the current four digit year
+
+## Syntax
+
+```js
+myDate.getFullYear()
+```
+
+## Example
+
+```js
+const getYear = new Date().getFullYear()
+// returns 2023
```

Comment:
```suggestion
```
This entry will remain up beyond the current year, so remove this comment. Also, add a line that does something with the result, such as print it to the console.